gap/utils.g

```diff
@@ -19,46 +19,72 @@ end;
 
 ##############################################################################
 # Larger high-level functions.
-GAUSS_ChopMatrix := function( f,A,nrows,ncols )
-    local   i, j, rrem, crem, rowCnt, rowOverhead, colOverhead, colCnt, AA, a,
-    b, rowsList, colsList;
+GAUSS_calculateBlockSize := function(i, j, nrVerticalBlocks, nrHorizontalBlocks,
```

Comment:
I think calculateBlockSize would become easier if you passed it `(i, j, verticalRemainder, horizontalRemainder, a, b)`. Currently you do `RemInt` and `QuoInt` for each combination of `i` and `j`. Instead you can do that once in ChopMatrix.

Also wouldn't `nrColsPerBlock` and `colOverhead` be better names for `horizontalOverhead` and `currentHorizontalOverhead` respectively?

Reply:
Remove
`nrVerticalBlocks, nrHorizontalBlocks, nrRowsTotal, nrColsTotal`